reference/6/Microsoft.PowerShell.Management/Clear-ItemProperty.md

```diff
@@ -39,9 +39,11 @@ You can use this cmdlet to delete the data from a registry value.
 
 ### Example 1: Clear the value of registry key
 
-This command clear the data in the "Options" registry value in the "MyApp" subkey of "HKEY_LOCAL_MACHINE\Software\MyCompany".
+This command clear the data in the "Options" registry value in the "MyApp" subkey of
+"HKEY_LOCAL_MACHINE\Software\MyCompany".
 
-Because the command is being issued from a file system drive (`C:`), it uses the fully qualified path to the `HKLM:` drive and the "Software\MyCompany\MyApp" subkey.
+Because the command is being issued from a file system drive (`C:`), it uses the fully qualified
```

Comment:
Since we are not showing the PS prompt I don't think this statement is needed. Using the full path works no matter what drive you are on. I don't think we need to say anything about that.